### PR TITLE
[R2] S3 API compatibility page

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: content/r2/platform/s3-compatibility.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+Feature implementation is currently in progress. Refer back to this page for updates.
+
+## How to read this page
+
+Each operations section will include a table of implemented and and another table of unimplemented APIs. Refer the feature column of each table to review which specific features of the API have been implemented and which are not.
+
+## Bucket level operations
+
+Below is a list of implemented bucket level operations. Refer to the Feature column to review which features have been implemented (✅) and have not (❌) been implemented.
```
Comment (WalshyDev) [inline content/r2/platform/s3-compatibility.md]: missing bracket
```suggestion
Below is a list of implemented bucket level operations. Refer to the Feature column to review which features have been implemented (✅)) and have not (❌) been implemented.
```

File: content/r2/platform/S3 Compatibility/_index.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
```
Comment (kodster28) [inline content/r2/platform/S3 Compatibility/_index.md]: We shouldn't have spaces in our URLs. I'd rename this folder to `s3-compatibility`

File: content/r2/platform/S3 Compatibility/s3-compatibility.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+  title: S3 API Compatibility
+---
+
+# S3 API compatibility
+
+Feature implementation is currently in progress. Refer back to this page for updates.
+
+## How to read this page
+
+Each operations section will include a table of implemented and a table of unimplemented APIs. Refer the feature column of each table to review which specific features of the API have been implemented and which are not.
```
Comment (kodster28) [inline content/r2/platform/S3 Compatibility/s3-compatibility.md]:
```suggestion
Each operations section will include two tables: one of implemented APIs and one of unimplemented APIs. Refer the feature column of each table to review which specific features of the API have been implemented and which have not.
```

File: content/r2/platform/S3 Compatibility/_index.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+title: S3 Compatibility
+pcx-content-type: navigation
+---
```
Comment (kodster28) [inline content/r2/platform/S3 Compatibility/_index.md]: This should have a directory listing component or... if it's just meant to be a standalone page, it should just be a markdown file named w/in the platform folder.

File: content/r2/platform/S3 Compatibility/s3-compatibility.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+## Bucket-level operations
+
+Below is a list of implemented bucket-level operations. Refer to the Feature column to review which features have been implemented (✅) and have not been implemented (❌).
+
+{{<table-wrap style="width:100%">}}
+
+| API Name                                                                                | Feature                           |
+| -------------------------------------------------------------------------------------   |---------------------------------- |
+| [ListBuckets](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListBuckets.html)     |                                    |
```
Comment (kodster28) [inline content/r2/platform/S3 Compatibility/s3-compatibility.md]: Sorta confused why we have this command but nothing to say about it?

File: content/r2/platform/S3 Compatibility/s3-compatibility.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+
+Each operations section will include a table of implemented and a table of unimplemented APIs. Refer the feature column of each table to review which specific features of the API have been implemented and which are not.
+
+✅ Feature Implemented  
+❌ Feature Not Implemented
+
+## Bucket-level operations
+
+Below is a list of implemented bucket-level operations. Refer to the Feature column to review which features have been implemented (✅) and have not been implemented (❌).
+
```
Comment (kodster28) [inline content/r2/platform/S3 Compatibility/s3-compatibility.md]: Is this meant to be the implemented table? If so, label it. If not, should be combined with table below.

File: content/r2/platform/S3 Compatibility/s3-compatibility.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+{{<table-wrap style="width:100%">}}
+
+| API Name                                                                                | Feature                           |
+| -------------------------------------------------------------------------------------   |---------------------------------- |
+| [ListBuckets](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListBuckets.html)     |                                    |
+| [HeadBucket](https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadBucket.html)       | ❌ Bucket Owner: <br> ❌ x-amz-expected-bucket-owner  |
+| [CreateBucket](https://docs.aws.amazon.com/AmazonS3/latest/API/API_CreateBucket.html)   | ❌ ACL: <br> ❌ x-amz-acl <br> ❌ x-amz-grant-full-control <br> ❌ x-amz-grant-read <br> ❌ x-amz-grant-read-acp  <br> ❌ x-amz-grant-write <br> ❌ x-amz-grant-write-acp |
```
Comment (kodster28) [inline content/r2/platform/S3 Compatibility/s3-compatibility.md]: Not sure I'm in love with the single column formatting for this (sorta hard to tell which is a top-level feature and which isn't).

Don't really know if we have too too many other options for formatting though?

File: content/r2/platform/S3 Compatibility/s3-compatibility.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+|                                                                                         | ❌ Request Payer: <br> ❌ x-amz-request-payer |
+|                                                                                         | ❌ Tagging: <br> ❌ x-amz-tagging |      |
+|                                                                                         | ❌ Object Locking: <br> ❌ x-amz-object-lock-mode <br> ❌ x-amz-object-lock-retain-until-date <br> ❌ x-amz-object-lock-legal-hold |
+|                                                                                         | ❌ ACL: <br> ❌ x-amz-acl <br> ❌ x-amz-grant-full-control <br> ❌ x-amz-grant-read <br> ❌ x-amz-grant-read-acp <br> ❌ x-amz-grant-write-acp |
+|                                                                                         | ❌ Bucket Owner: <br> ❌ x-amz-expected-bucket-owner |
```
Comment (kodster28) [inline content/r2/platform/S3 Compatibility/s3-compatibility.md]: Storage class?

File: content/r2/platform/S3 Compatibility/s3-compatibility.md (hunk 7 above)
Comment (kodster28) [reply content/r2/platform/S3 Compatibility/s3-compatibility.md]: I think it's mostly just confusing having a check next to the category name (`Operation Metadata`, `System Metadata`) in addition to the sub-items (especially when they're the same level of indentation).

**Review by kodster28 — APPROVED** [review]

**Pull request merged**